it/uk/gov/hmrc/soletraderidentificationfrontend/views/CaptureDateOfBirthViewTests.scala

```diff
@@ -29,6 +30,15 @@ trait CaptureDateOfBirthViewTests {
 
   def testCaptureDateOfBirthView(result: => WSResponse): Unit = {
     lazy val doc: Document = Jsoup.parse(result.body)
+    lazy val config = app.injector.instanceOf[AppConfig]
+
+    "have a sign out link in the header" in {
+      doc.getSignOutText mustBe Header.signOut
+    }
+
+    "sign out link redirects to feedback page" in {
```

Comment:
This test reads as 'should sign out link redirects to feedback page'. Please could you change it so it reads better. Will need to do the other tests as well